neqo-bin/src/udp.rs

```diff
@@ -56,10 +57,12 @@ impl Socket {
     /// Receive a batch of [`Datagram`]s on the given [`Socket`], each set with
     /// the provided local address.
     pub fn recv(&self, local_address: &SocketAddr) -> Result<Vec<Datagram>, io::Error> {
+        let mut recv_buf = vec![0; RECV_BUFFER_SIZE];
         self.inner
             .try_io(tokio::io::Interest::READABLE, || {
-                neqo_udp::recv_inner(local_address, &self.state, &self.inner)
+                neqo_udp::recv_inner(local_address, &self.state, &self.inner, &mut recv_buf)
             })
+            .map(|dgrams| dgrams.map(|d| d.to_owned()).collect())
```

Comment:
Will be reverted in final stacked pull request https://github.com/mozilla/neqo/pull/2189. Placeholder here in order to fully compile and pass tests.

Reply:
With https://github.com/mozilla/neqo/pull/2189 merged, this is resolved.